Recrutify/Recrutify.Services/Validators/CreateCandidateValidator.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Linq;
+using FluentValidation;
+using Recrutify.Services.DTOs;
+
+namespace Recrutify.Services.Validators
+{
+    public class CreateCandidateValidator : AbstractValidator<CandidateCreateDTO>
+    {
+        public CreateCandidateValidator()
+        {
+            RuleFor(c => c.Name)
+                .NotNull()
+                .NotEmpty()
+                .MaximumLength(50);
+            RuleFor(c => c.Surname)
+                .NotNull()
+                .NotEmpty()
+                .MaximumLength(50);
+            RuleFor(c => c.PhoneNumber)
+                .NotNull()
+                .NotEmpty()
+                .Matches(@"^\+?(\d[\d\- ]+)?(\(?[\d\- ]+\))?([\d\- ])+\d$");
+            RuleFor(c => c.Email)
+                .NotNull()
+                .NotEmpty()
+                .EmailAddress();
+            RuleFor(c => c.Contacts)
+                .NotNull()
+                .NotEmpty();
+            RuleFor(c => c.Contacts).
+                Must(c => c.Any(contact => contact.Type == "Skype"))
```

Comment:
move Skype to constant field of validator class

Reply:
String Skype?

Reply:
yes

Reply:
and move dot to next lune from 30 to 31